main.py

```diff
@@ -282,6 +283,67 @@ async def on_message_edit(before, after):
             ctx = await bot.get_context(after)
             await bot.invoke(ctx)
 
+@bot.event
+async def on_message(message):
+	if bot.afk_users.get(message.author.id):
```

Comment:
Indentation errors here

Reply:
Ah, do you use spaces for indentation?

Reply:
Of course, as per https://www.python.org/dev/peps/pep-0008/#code-lay-out.
Regardless, there are inconsistencies whether or not spaces or tabs are used.